lndclient/router_client.go

```diff
@@ -49,6 +47,32 @@ type SendPaymentRequest struct {
 	MaxCltv         *int32
 	OutgoingChannel *uint64
 	Timeout         time.Duration
+
+	// Target is the node in which the payment should be routed towards.
```

Comment:
maybe note that all these will be ignored if the invoice is non-empty

Reply:
or return an error if both are set.

Reply:
comment added

Reply:
I mean, it should be noted on the fields on `SendPaymentRequest`

Reply:
Added